SuperChamps/contracts/Metagame/SCMetagameHouseRewards.sol

```diff
@@ -50,10 +53,59 @@ contract SCMetagameHouseRewards is StakingRewards {
         require(access_pass.isVerified(msg.sender), "MUST HAVE VERIFIED ACCESS PASS");
         require(amount_ > 0, "CANNOT STAKE 0");
 
-        _totalSupply = _totalSupply + amount_;
+        uint256 _old_multiplier_basis_points = _multiplier_basis_points[msg.sender];
+        _multiplier_basis_points[msg.sender] = access_pass.getMetagameMultiplier(msg.sender);
+
+        _totalSupply -= _balances[msg.sender] * _old_multiplier_basis_points / 10000;
```

Comment:
@figs999 whats the logic for division by 10000 everywhere?

Reply:
The multiplier is in basis points.